train.py

```diff
@@ -75,22 +51,24 @@ def save_model(model, epoch):
 	}
 	torch.save(model_state, filename)
 
-def train(num_epochs, model, train_loader,test_loader, optimizer, criterion, device,log_interval):
+def train(num_epochs, model, train_loader,test_loader, optimizer, criterion, device,log_interval,vocab):
 	"""
 	Executes model training. Saves model to a file every epoch.
```

Comment:
we should modify the description of the function here since we don't "save the model to a file every epoch"

Reply:
Done!